sci-mathematics/normaliz/normaliz-3.8.9.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools toolchain-funcs eutils
+
+MYP="Normaliz-${PV}"
+
+DESCRIPTION="Tool for computations in affine monoids and more"
+HOMEPAGE="http://www.mathematik.uni-osnabrueck.de/normaliz/"
+SRC_URI="https://github.com/Normaliz/Normaliz/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0/3"
+KEYWORDS="~amd64 ~arm ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc extras openmp static-libs"
+
+# would be nice to package scip and cocoalib
+
+RDEPEND="
+	dev-libs/gmp:=[cxx]
+"
+DEPEND="${RDEPEND}
+	dev-libs/boost
+"
+# Only a boost header is needed -> not RDEPEND
```

Comment:
Should it use := to use the new header when new boost is installed?

Reply:
I'm not even sure but you are probably right.
Just let this sit for a few days, my brain is fried.
Dumb package broke backwards compatibility so I'm not even sure if we want to add a new subslot or not...

Reply:
@DarthGandalf from my quick glance, it only uses C++ headers, and doesn't link, so this should be fine